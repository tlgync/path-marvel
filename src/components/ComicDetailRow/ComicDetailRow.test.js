import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { ComicDetailRow } from '.';

test('ComicDetailRow Render SnapShot', () => {
  const title = 'Page Count';
  const content = '32';
  const tree = renderer.create((
    <Router>
      <ComicDetailRow title={title} content={content} />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
