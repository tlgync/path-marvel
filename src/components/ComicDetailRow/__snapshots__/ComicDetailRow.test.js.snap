// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComicDetailRow Render SnapShot 1`] = `
<div
  className="comic-title-row"
>
  <span>
    Page Count
    : 
  </span>
  <span>
    32
  </span>
</div>
`;
